### DISCUSSION TITLE
Payment information

### DISCUSSION THREAD

**Discussion by Donglethaoly1106** [community/community#146584] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I don't remember if I recorded my bank card on github, but I didn't buy any product package. Now I'm charged, how can I remove my bank account from github and keep my current package that has been activated?

**Comment by nazifkaraca** [comment]
Hello there!

Following these steps may help you:
- Click on your profile photo in the upper-right corner, then select Settings.
- In the "Access" section of the sidebar, click Billing and plans, then Plans and usage.
- Here, you can view your current plan and any active subscriptions.
- Within the Billing and plans section, locate the Payment information.
- Click Edit next to your payment method.
- To remove your payment method, click Remove.

As far as I know, you don't need to do anything for your current package. **It will continue until the end of your subscription**!

If this comment helps, please upvote my comment! 

Have a nice day

**Comment by mecodeatlas** [comment]
Hi @Donglethaoly1106! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

Oh, and you are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you :).
